src/NUnitTestExtractor/NUnitTestExtractor.Tests/NUnitTestExtractorTests.cs

```diff
@@ -3,22 +3,309 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.IO;
 using NUnit.Framework;
+using NUnitTestExtractor;
+using System.Diagnostics;
+using System.Collections;
 
-namespace NUnitTestExtractor.Tests
+namespace NUnitTestExtractorTests
 {
     [TestFixture]
     public class NUnitTestExtractorTests
     {
-        [Test]
-        public void DirectoryCreatedWhenOutput()
+        private const string outputDirectory = "C:/Git/NTestController/src/NUnitTestExtractor/NUnitTestExtractor.Tests/bin/Debug/TestOutput/file.txt";
+        
+        /// <summary>
+        /// Used to invoke the GetTests() method in NUnitTestExtractorApp
+        /// </summary>
+        /// <param name="dlls">The dlls to pass to GetTests()</param>
+        /// <param name="level">The level to pass to GetTests()</param>
+        private void InvokeGetTestsUsingFileOutput(List<string> dlls, NUnitTestExtractorApp.Level level)
         {
+            using (StreamWriter writer = new StreamWriter(outputDirectory))
```

Comment:
Maybe instead of writing to a file, you can write to a MemoryStream so you don't have to worry about paths or filesystems at all, and it should be faster than writing to disk.
(That's one of the advantages of having GetTests() take a StreamWriter)
You might also be able to change GetTests() to take a TextWriter instead of a StreamWriter and then you can use a StringWriter here.
https://stackoverflow.com/questions/235156/redirect-net-streamwriter-output-to-a-string-variable